srcpkgs/python3-halo/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'python3-halo'
+pkgname=python3-halo
+version=0.0.31
+revision=1
+wrksrc="halo-${version}"
+build_style=python3-module
+hostmakedepends="python3-pip python3-wheel"
```

Comment:
```suggestion
hostmakedepends="python3-setuptools"
```

Reply:
I can't tell you why, but this package won't build without both pip and wheel. Same goes for the others where I have `"python3-pip python3-wheel"` as hostmakedepends.

Reply:
Thanks to @paper42's help on IRC this now resolved for all packages :)